src/share/classes/com/sun/javatest/regtest/exec/RegressionScript.java

```diff
@@ -1123,7 +1128,7 @@ Path getJavacProg() {
 
     Map<String, String> getTestProperties()  {
         // initialize the properties with standard properties common to all tests
-        Map<String, String> p = new LinkedHashMap<>(params.getBasicTestProperties());
+        Map<String, String> p = new TreeMap<>(params.getBasicTestProperties());
```

Comment:
This change sorts all basic test properties by their name; keeping the insertion order was never a promise.

Reply:
Does the new proposed feature of configuring a default timeout continue to work without this change to using a `TreeMap`? If yes, then I think it would be good to separate this change out from this PR.

Reply:
Good point. Reverting this change.